src/adapters/MeetingsJSONAdapter/controls/DisabledJoinControl.js

```diff
@@ -0,0 +1,36 @@
+import {Observable} from 'rxjs';
+import {MeetingControlState} from '@webex/component-adapter-interfaces';
+
+/**
+ * Display options of a meeting control.
+ *
+ * @external MeetingControlDisplay
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
+ */
+
+export default class DisabledJoinControl {
+  constructor(adapter, controlID) {
```

Comment:
let's move this constructor in a parent base class called `MeetingControl` and extend each control from the base class.